python/neuroglancer/pipeline/task_queue.py

```diff
@@ -247,15 +276,16 @@ def purge(self):
 
             for tid in taskids:
                 self.delete(tid)
+            
```

Comment:
I feel like a purge by tag would be pretty useful, even better if we make our tag system more powerful